pkg/volume/util/atomic_writer_test.go

```diff
@@ -390,7 +385,7 @@ IAAAAAAAsDyZDwU=`
 	}
 
 	for _, tc := range cases {
-		targetDir, err := utiltesting.MkTmpdir("atomic-write")
+		targetDir, err := os.MkdirTemp("", "atomic-write")
```

Comment:
I'm not sure this case is special enough to make an exception. Please, use `t.TempDir()`

Reply:
done